docs/user/security/api-keys/index.asciidoc

```diff
@@ -74,7 +74,7 @@ The response should look something like this:
 
 Now, you can use the API key to request {kib} roles. You will need
 to base64-encode the `id` and `api_key` provided in the response
-and add it to your request as an authorization header. For example:
+and add it to your request as an authorization header, under the format `id + ":" + api_key`. For example:
```

Comment:
__nit:__ I'd borrow the text we already use in the [Elasticsearch docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-api-create-api-key.html), just for the consistency sake. Something like this:

> Now, you can use the API key to request {kib} roles. You'll need to send a request with a `Authorization` header with a value having the prefix `ApiKey` followed by the credentials, where credentials is the base64 encoding of `id` and `api_key` joined by a colon. For example:

@gchaps what do you think?

Reply:
Yes, best to use the same wording as in the Elasticsearch docs.

Reply:
Thanks! @theo2354 would you mind applying the latest suggestion then?

Reply:
I have applied the suggestion, thank you.